web-ui/src/main/resources/catalog/locales/en-admin.json

```diff
@@ -1399,6 +1399,10 @@
     "ui-fluidLayout-help": "If enabled, the layout of the application has a full width container, when disabled the layout has a fixed width and centered container",
     "ui-fluidEditorLayout": "Fluid container for the Editor",
     "ui-fluidEditorLayout-help": "If enabled, the layout of the application has a full width container, when disabled the layout has a fixed width and centered container",
+    "ui-fluidHeaderLayout": "Fluid container for Header and Top menu",
+    "ui-fluidHeaderLayout-help": "If enabled, the layout of the header and top menu has a full width container, when disabled the layout has a fixed width and centered container",
+    "ui-showGNName": "Show the GeoNetwork name in the Top menu",
```

Comment:
Maybe `Show the catalogue name in the top menu` ?

Reply:
Yes, I wanted to add an option for this in a new PR. But I can also add it to this one.

Reply:
Aha, now I understand your remark. I will change it.